public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -468,11 +468,76 @@ var skillsHelpURL = {
   'summon familiar'  :        baseHelpURL + "classes/sorcererWizard.htm#wizardFamiliar",
   'scribe scroll'    :        baseHelpURL + "classes/sorcererWizard.htm#scribeScroll",
 
+  // ******************
+  // Sidequestors Links
+  // ******************
+
+  // Feats:
+  'expeditious dodge':        'http://sidequestors.org/wiki/Expeditious_Dodge',
+
+  // Dervish special abilities.
+  'dervish dance'    :        'http://sidequestors.org/wiki/Dervish#Dervish_Dance_.28Ex.29',
+  'movement mastery' :        'http://sidequestors.org/wiki/Dervish#Movement_Mastery_.28Ex.29',
+  'slashing blades'  :        'http://sidequestors.org/wiki/Dervish#Slashing_Blades',
+  'dance of death'   :        'http://sidequestors.org/wiki/Dervish#Dance_of_Death',
+  'improved reaction':        'http://sidequestors.org/wiki/Dervish#Improved_Reaction_.28Ex.29',
+  'elaborate parry'  :        'http://sidequestors.org/wiki/Dervish#Elaborate_Parry_.28Ex.29',
+  'tireless dance'   :        'http://sidequestors.org/wiki/Dervish#Tireless_Dance',
+  'a thousand cuts'  :        'http://sidequestors.org/wiki/Dervish#A_Thousand_Cuts_.28Ex.29',
+
+  // Radiant servant of Pelor special abilities.
+  'extra greater turning' :   'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Extra_Greater_Turning',
+  'radiance'         :        'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Radiance',
+  'empower healing'  :        'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Empower_Healing_.28Ex.29',
+  'aura of warding'  :        'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Aura_of_Warding_.28Su.29',
+  'maximize healing' :        'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Maximize_Healing_.28Ex.29',
+  'positive energy burst' :   'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Positive_Energy_Burst_.28Su.29',
+  'supreme healing'  :        'http://sidequestors.org/wiki/Radiant_servant_of_Pelor#Supreme_Healing_.28Ex.29',
+
+
   '###last one###'   :           ""
 };
 
 
 var spellsHelpURL = {
+  // ************************
+  // Begin Sidequestors Links
+  // ************************
+  "blessed aim":"http://sidequestors.org/wiki/Blessed_aim",
+  "guiding light":"http://sidequestors.org/wiki/Guiding_light",
+  "light of Lunia":"http://sidequestors.org/wiki/Light_of_Lunia",
+  "vigor, lesser":"http://sidequestors.org/wiki/Lesser_vigor",
```

Comment:
eccentric placement; why not with the other vigor variants ?  Harmless.

Generally, this list shall be easier to maintain if kept in alphabetic order, except for putting prefixes after everything of which they're a prefix (i.e. sorting end-of-name after every character).